Tasks/Ilyasevich/Task5/Task5.py

```diff
@@ -0,0 +1,65 @@
+def check_str(string_input):
+    """ пробегаем по всей строке и возвращаем для указанного Юникод-символа число, представляющее его позицию кода """
+    lowercase_letter = []
+    uppercase_letter = []
+    for i in range(len(string_input)):
+        if ord(string_input[i]) in range(65, 91):
+            uppercase_letter.append(string_input[i])
+        if ord(string_input[i]) in range(97, 123):
+            lowercase_letter.append(string_input[i])
```

Comment:
consider using specific function **isupper** and **islower** instead of a direct check for the symbol number